### PR TITLE
v1.0.0 Bugfix and code cleanup

### PR DESCRIPTION
**Pull request by miketalley**
- Split up `commands/create`
  - Moved `onClickCreateFolder` and `onClickCreateFile` to /lib/fileHelpers`
  - Moved serverless commands to `commands/serverless`
  - Moved template commands to `commands/templates` 
- Fixed `AccountTreeItem` issue and removed `@ts-ignore`
- Updated some uses of `import * as ...`, opting for importing only specific methods
- Removed unused variables

### CODE REVIEW

File: src/lib/commands.ts [hunk 1]
```diff
@@ -14,8 +15,9 @@ export const registerCommands = (
   registerAccountCommands(context);
   registerAuthCommands(context, rootPath);
   registerConfigCommands(context);
+  registerGlobalStateCommands(context);
   registerNotificationCommands(context);
-  registerCreateCommands(context);
+  registerServerlessCommands(context);
+  registerTemplateCommands(context);
   registerTerminalCommands(context);
-  registerGlobalStateCommands(context);
```
Comment (miketalley) [inline src/lib/commands.ts:20]: Removing create, adding serverless and template, and reordering alphabetically.

File: src/lib/commands/serverless.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            endpointPath: 'serverless',
+            endpointMethod: 'GET',
+          },
+          dir
+        );
+        trackEvent(TRACKED_EVENTS.CREATE.SERVERLESS_FUNCTION_FOLDER);
+      })
+    )
+  );
+};
```
Comment (miketalley) [inline src/lib/commands/serverless.ts:67]: Moved these over from /commands/create and updated imports for `path`

File: src/lib/commands/templates.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          basename(uniqueFilePath),
+          dirname(uniqueFilePath),
+          TEMPLATE_NAMES.GLOBAL_PARTIAL,
+          { allowExisting: true }
+        );
+        trackEvent(TRACKED_EVENTS.CREATE.GLOBAL_PARTIAL);
+      })
+    )
+  );
+};
```
Comment (miketalley) [inline src/lib/commands/templates.ts:69]: Moved these over from /commands/create and updated imports for `path`

File: src/lib/fileHelpers.ts [hunk 4]
```diff
@@ -26,3 +34,94 @@ export const getUniquePathName = (folderPath: string, extension: string) => {
...
+    if (clickContext.scheme === 'file') {
+      const createFileSubscription = workspace.onWillCreateFiles(
+        buildFolderCreateListener(extension, clickContext.fsPath, doAfter, () =>
+          createFileSubscription.dispose()
+        )
+      );
+      commands.executeCommand('explorer.newFolder');
+    }
+  };
+};
```
Comment (miketalley) [inline src/lib/fileHelpers.ts:127]: Moving these over from `/commands/create`

File: src/lib/terminal.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
-import { commands, ExtensionContext } from 'vscode';
+import { commands } from 'vscode';
 import { COMMANDS, POLLING_INTERVALS } from './constants';
 
-export const initializeTerminal = (context: ExtensionContext) => {
```
Comment (miketalley) [inline src/lib/terminal.ts:4]: Not being used here

**Review by TanyaScales — APPROVED** [review]

**Review by jsines — APPROVED** [review]

**Pull request merged**